src/deploy/project.py

```diff
@@ -239,8 +239,14 @@ def is_release_deployed(self, release, environment_id):
                 deployed_tasks = []
                 for task in tasks:
                     task_tags = parse_aws_tags(task["tags"])
-                    task_deployment_label = task_tags["deployment:label"]
-                    deployed_tasks.append(task_deployment_label == service_deployment_label)
+
+                    _is_task_deployed = (
+                        task_tags.get("deployment:label") == service_deployment_label
+                    ) and (
+                        task['lastStatus'] == "RUNNING"
```

Comment:
We should check the task status, as an edge case can occur where deployments are marked as good before they have started . See https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-lifecycle.html.

I have seen this issue "in the wild" where containers cannot start because they have bad permissions or there is a problem with the host instance (if you have an EC2 host).